source/workflow/requirements.txt

```diff
@@ -1 +1 @@
-../../source/lib/MediaInsightsEngineLambdaHelper/dist/Media_Insights_Engine_Lambda_Helper-0.0.3-py3-none-any.whl
+
```

Comment:
Was this just a stale artifact?

Reply:
I added the lambda layer to this lambda so it doesn't need to package the lambda helper anymore.

Reply:
👍